test/CreateTransactionTest.php

```diff
@@ -14,26 +14,42 @@ class CreateTransactionTest extends \PHPUnit_Framework_TestCase
 
     /**
      * @covers  Islandora\Chullo\Chullo::createTransaction
+     * @covers  Islandora\Chullo\Chullo::getBaseUri
      * @covers  Islandora\Chullo\FedoraApi::createTransaction
+     * @covers  Islandora\Chullo\FedoraApi::prepareUri
+     * @covers  Islandora\Chullo\FedoraApi::getBaseUri
+     * @covers  Islandora\Chullo\FedoraApi::createTransaction
+     * @covers  Islandora\Chullo\FedoraApi::generateTransactionUri
      * @uses    GuzzleHttp\Client
      */
     public function testReturnsIdOn201()
     {
         $mock = new MockHandler([
             new Response(201, ['Location' => "http://localhost:8080/fcrepo/rest/tx:abc-123"]),
+            new Response(201, ['Location' => "http://localhost:8080/fcrepo/rest/tx:abc-123"]),
         ]);
 
         $handler = HandlerStack::create($mock);
         $guzzle = new Client(['handler' => $handler, 'base_uri' => 'http://localhost:8080/fcrepo/rest']);
         $api = new FedoraApi($guzzle);
         $client = new Chullo($api);
 
+        $this->assertEquals($client->getBaseUri(), 'http://localhost:8080/fcrepo/rest');
```

Comment:
I would be more concerned, that getBaseUri is [already covered](https://codecov.io/gh/Islandora-CLAW/chullo/src/998fd3e105c912c67d9c01f07d92a29b8ada6ce9/src/Chullo.php#L57) so either this is not a concern anymore or you have un-covered it. Maybe you need a rebase?